sklearn/metrics/_ranking.py

```diff
@@ -1115,6 +1115,10 @@ def roc_curve(
         fpr and tpr. `thresholds[0]` represents no instances being predicted
```

Comment:
> I still feel that the whole description of the thresholds attribute in the roc_curve can be improved to avoid redundancies.

I think your description is clearer than this one. Whey don't we the wording of your description (and maybe make it explicit it is `threshold[0]`), then in the `versionchanged` we can just say something like: "The arbitrary threshold at infinity is added."